code/datums/elements/entered_open_space_fall.dm

```diff
@@ -0,0 +1,42 @@
+/*
+ * An element specific to turfs that can drop movables on the z level beneath when moved onto.
+ * Basically, we want the fall to happen in [/atom/movable/proc/Moved()], not in [/turf/Exited()].
+ * Should make pulling, riding vehicles etc. through z levels a bit easier without resorting to many hacks and snowflakes.
+ */
+/datum/element/entered_open_space_fall
```

Comment:
this name is a bit weird, but I don't necesarily have a good name either. this one is a bit too over-complex while not actually explaining what the element does however.